spec/PhpSpec/Wrapper/Subject/Expectation/ConstructorDecoratorSpec.php

```diff
@@ -22,35 +22,20 @@ function let(Expectation $expectation)
 
     function it_rethrows_php_errors_as_phpspec_error_exceptions(Subject $subject, WrappedObject $wrapped)
```

Comment:
I think you have to amend the title because you are throwing now PhpSpec exception instead of Php Error